pywt/_multidim.py

```diff
@@ -276,7 +277,7 @@ def idwtn(coeffs, wavelet, mode='symmetric', axes=None):
     return coeffs['']
 
 
-def swt2(data, wavelet, level, start_level=0):
+def swt2(data, wavelet, level, start_level=0, axes=(-2, -1)):
```

Comment:
True.  It is only really there for backwards compatability.  I cannot just set `swt2 = swtn`, though, as the coefficient format returned is not the same.  The same thing is True of `dwt2`/`dwtn` as well.